pyphare/pyphare/pharesee/geometry.py

```
@@ -144,12 +157,122 @@ def periodicity_shifts(domain_box):
         )
 
     if domain_box.ndim == 3:
-        raise ValueError("Unhandeled dimension")
+        front = {
+            f"{k}front": [(v[0], v[1], shape_z) for v in l] for k, l in shifts.items()
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)